services/cinema_hall.py

```diff
@@ -0,0 +1,20 @@
+from typing import List
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> List[CinemaHall]:
```

Comment:
Please change the return annotation to queryset of CinemaHall instead of List[CinemaHall].